pkg/azurePipelines/azurePipelines_test.go

```diff
@@ -148,7 +162,7 @@ func newDraftConfig() *config.DraftConfig {
 			{
 				Name: "MANIFESTPATH",
 				Default: config.BuilderVarDefault{
-					Value: "manifests",
+					Value: "test/manifests",
```

Comment:
any reason why `test` dir was added?

Reply:
It was added to match the manifests path in the fixtures directory.